plutus-tx-plugin/plutus-tx-plugin.cabal

```diff
@@ -109,4 +109,5 @@ test-suite plutus-tx-tests
         lens -any,
         ghc-prim -any,
     -- this makes the plugin give better errors
-    ghc-options: -g
+    -- NOTE: we are disabling the simplifier
```

Comment:
why?

Reply:
added a comment that we need to disable the simplifier as it will simplify away some tests.